src/Raven.Server/Config/Categories/MemoryConfiguration.cs

```diff
@@ -20,6 +20,8 @@ public MemoryConfiguration()
             UseTotalDirtyMemInsteadOfMemUsage = PlatformDetails.RunningOnDocker;
 
             EnableHighTemporaryDirtyMemoryUse = false; //MemoryInformation.TotalPhysicalMemory.GetValue(SizeUnit.Gigabytes) >= 2;
+
+            MinSwapSize = MemoryInformation.TotalPhysicalMemory;
```

Comment:
Isn't it too much? The recommendation in the issue (https://issues.hibernatingrhinos.com/issue/RavenDB-15524) was "half the size of the memory?".  

We don't have it configured in our cloud this way. @gregolsky told me we have 1 - 2 GB of swap. So especially on stronger machines we'll be getting that alert.

CC @grisha-kotler

Reply:
In windows, I was a recommendation for 1.5X of the physical memory
And in Linux I found more detailed recommendation:
Physical | Swap 
  <2        | X2
 2-8        | X1
 8-64      | X1/2

So as constant ratio default the most reasonable value to me was X1 
But actually we can use a dynamic ratio

Reply:
Note that this is about minimal swap size, not the recommended size. Unless we want to change that alert to recommend the swap size instead of alerting on the minimal.

Since we're setting the default value dynamically it doesn't have to be constant. You can calculate it dynamically based on the platform and available physical memory size.